api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -211,12 +210,9 @@ message HttpHeaders {
   config.core.v3.HeaderMap headers = 1;
 
   // [#not-implemented-hide:]
-  // TODO(jbohanon) reserve field as part of rework detailed in https://github.com/envoyproxy/envoy/issues/32125
-  // The values of properties selected by the ``request_attributes``
-  // or ``response_attributes`` list in the configuration. Each entry
-  // in the list is populated
-  // from the standard :ref:`attributes <arch_overview_attributes>`
-  // supported across Envoy.
+  // This field is deprecated and not implemented. Attributes will be sent in
+  // the  top-level :ref:`attributes <envoy_v3_api_field_service.ext_proc.v3.ProcessingRequest.attributes`
+  // field.
   map<string, google.protobuf.Struct> attributes = 2;
```

Comment:
Add the deprecated annotation to this field `[deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"]`